apps/server/src/modules/authorization/domain/rules/index.ts

```diff
@@ -15,3 +15,4 @@ export * from './team.rule';
 export * from './user-login-migration.rule';
 export * from './user.rule';
 export * from './group.rule';
+export { SystemRule } from './system.rule';
```

Comment:
Okay, but i think the requirement of explicit exports is for the barrel file of the module only

Reply:
there is no documentation on how to do it only the comment from cedric that exports have to be explicit

Reply:
yes, i know. was just a hint :P